packages/OreIdProfile/package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "oreid-profile",
+  "version": "1.0.1",
+  "license": "MIT",
+  "main": "dist/index.js",
+  "source": "src/index.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "rm -rf dist/ && rollup -c",
+    "prepublish": "yarn build",
+    "dev": "rollup -c -w",
+    "test": "jest",
```

Comment:
There are no tests written.

Reply:
Done

Reply:
Really, but I chose to keep it because my intention is to put the tests in the next sprints. I just haven't put it yet because I believe that tray will ask for several changes.

Anyway, I left the tests there to emphasize the need of them.

Reply:
Sounds good.